### PR TITLE
EuphonyTxPanel Component & Documentation

### PR DESCRIPTION
**Pull request by zion830**
Related issue :  #231 

* It is an UI component for easy use of `EuTxManager`. It also provides several options to custom design.
* And I added a new markdown file to introduce Euphony Ui

### Screenshot
eutxpanel_screenshot

### CODE REVIEW

File: euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    hint: String = "",
+    textColor: Color = Black,
+    hintTextColor: Color = LightGray,
+    textFieldBackgroundColor: Color = LightSkyBlue,
+    buttonBackgroundColor: Color = LightBlue,
+    panelHeight: Dp = 54.dp,
+    iconTintColor: Color = White
+) {
+    // TODO : EuTxManager() should be changed to use singleton pattern
+    val viewModel: TxPanelViewModel = viewModel(factory = TxPanelViewModelFactory(EuTxManager()))
```
Comment (zion830) [inline euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt:40]: If #255 is merged, I need to change this line

File: euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(panelHeight),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        TextField(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxHeight(),
```
Comment (yeonns) [inline euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt:56]: I tested your `EuphonyTxPanel`, and even if row has `panelHeight`, your `TextField` and `Button` has `fillMaxHeight()`, so it fills whole screen. So, I think `TextField` and `Button` should have panelHeight.

**Review by yeonns — COMMENTED** [review]
@zion830 Thank you for implementing EuphonyTxPanel!
Your test code and documentation is cool :+1: 
I commented one thing and check please :)

File: euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt (hunk 2 above)
Comment (zion830) [reply euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt]: Thank you for letting me know! I fixed it
[15c0d37](https://github.com/euphony-io/euphony/pull/262/commits/15c0d37cbed82765754fa82a5751a96f574f9c15)

File: euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt (hunk 1 above)
Comment (zion830) [reply euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt]: I applied the singleton pattern patch! [9be03e5](https://github.com/euphony-io/euphony/pull/262/commits/9be03e511f284a02a266ad9d9ab8027659c2b4a5)

**Pull request merged**